### PR TITLE
Add NFC support to TPOS Extension

### PR DESCRIPTION
**Pull request by leesalminen**
This PR adds NFC support to the TPOS extension, enabling payment to be collected via NFC tags that contain LNURLw strings (both raw and SUN authenticated via LNBits Bolt Cards extension).

### CODE REVIEW

**Comment by leesalminen** [comment]
A Twitter user checked out this branch and is using it for a live event right now. Got some feedback from them:

```We've so far made 708 real life payments with your TPOS NFC plugin since Thursday. Going strong.```

**Comment by leesalminen** [comment]
I'm ready for review and merge on this :).

File: lnbits/extensions/tpos/views_api.py [hunk 1]
```diff
@@ -79,6 +81,66 @@ async def api_tpos_create_invoice(
...
+            status_code=HTTPStatus.NOT_FOUND, detail="TPoS does not exist."
+        )
+
+    lnurl = (
+        lnurl_data.lnurl.replace("lnurlw://", "")
+        .replace("lightning://", "")
+        .replace("LIGHTNING://", "")
+        .replace("lightning:", "")
+        .replace("LIGHTNING:", "")
+    )
```
Comment (callebtc) [inline lnbits/extensions/tpos/views_api.py:103]: the things we have to deal with...

**Comment by callebtc** [comment]
LGTM, can't test though, so I'll merge?

**Comment by leesalminen** [comment]
Yes, I've tested myself and a third party user as well. LGTM!

**Pull request merged**